src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,61 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static final String CUSTOM_DELIMITER_PATTERN = "//(.)\n(.*)";
```

Comment:
Pattern 타입으로 멤버를 선언해도 좋을 것 같아요.
클래스 내에서만 사용된다면 접근 범위도 private으로 낮추면 어떨까요?
```suggestion
    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
```